app/controllers/admin/dashboard_controller.rb

```diff
@@ -30,6 +30,7 @@ def index
       @trending_hashtags     = TrendingTags.get(7)
       @profile_directory     = Setting.profile_directory
       @timeline_preview      = Setting.timeline_preview
+      @antispam_enabled      = Setting.antispam_enabled
```

Comment:
If the Ruby class is called "spam_check" would it not be more consistent to name it "spam_check_enabled"? That being said it probably doesn't matter.